spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDFactory.java

```diff
@@ -80,7 +90,7 @@ public RDD<InternalRow> buildScanFromSQL(String sql) {
 
```

Comment:
Is this method used only for query pushdowns or also for view materialization? We should change the label to `query_source` and the values can be "view_materialization" and "spark_query_pushdown"

Reply:
This one is only used for query pushdowns. But I will change the label to be `query_source`